template/.solidarity

```diff
@@ -67,24 +67,19 @@
       }
     ],
     "React Native": [
-      {
-        "rule": "cli",
-        "binary": "react-native",
-        "semver": ">=2.0.1"
-      },
-      {
-        "rule": "cli",
-        "binary": "react-native",
-        "semver": "^0.61.1",
-        "line": 2,
-        "error": "react-native@{{wantedVersion}} is required (you have {{installedVersion}}). Check your `package.json` and run `yarn install`."
+     {
+        "rule": "shell",
+        "command": "which react-native",
+        "match": "node_modules/.bin/react-native",
+        "error": "It looks like you have react-native-cli installed globally. Use 'yarn global remove react-native' or 'npm uninstall -g react-native'. See: 'https://github.com/react-native-community/cli#about'"
       }
     ],
     "Detox": [
       {
         "rule": "cli",
         "binary": "applesimutils",
         "semver": ">=0.7.4",
+        "error": "Use --fix. To upgrade, use 'brew upgrade applesimutils'. If you get errors, try 'brew untap wix/brew && brew tap wix/brew'",
```

Comment:
Maybe we could clarify the `Use --fix` part? From quick glance, I'm not sure what this means. Do you mean run `node ./setup --fix`, `applesimutils --fix` or something else?

Reply:
sure thing. it's built into solidarity, so perhaps `To install, use yarn solidarity --fix`.

Reply:
Sounds good to me! Once it's added in, re-request the review and I'll approve 👍